### ISSUE TITLE
How to typecast result from values method

### ISSUE THREAD

**Issue by abhi747** [jakearchibald/idb-keyval#104]
This is a query.

I am using TypeScript and want to type the return type of a function using `idb-keyval` method - `values`. But I am getting error saying `Argument of type 'IDBValidKey[]' is not assignable to parameter of type X[]`. Here `X` is a custom object that is saved in idb key-val storage in this case.

How to handle this?

**Comment by jakearchibald** [comment]
Ohh, this is a bug with a the types. Good catch.

**Comment by abhi747** [comment]
Ohh, I see.

For now, I have handled like below -

```
const getAllocationsInQueue = async (): Promise<_allocationRecord[]> => {
    const allocations = (await values()) as unknown;
    return allocations as _allocationRecord[];
}
```

**Comment by jakearchibald** [comment]
Fix published in 5.0.3
